src/_sass/footer2.scss

```diff
@@ -0,0 +1,152 @@
+.siteFooter2 {
+  width: 100%;
+  display: none;
+  margin-right: auto;
+  margin-left: auto;
+  
+  @include mobileAndAbove {
+    max-width: $tilt2-footer-width;
+    padding-top: $spacing-unit * 2;
+    padding-bottom: $spacing-unit * 2;
+  }
+}
+body.is-footer2 .siteFooter2 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.siteFooter2-links {
+  display: flex;
+
+  @include mobileAndBelow {
+    flex-direction: column;
+  }
+  @include mobileAndAbove {
+    margin-bottom: $spacing-unit;
+  }
+}
+.siteFooter2-links-navList {
+  @include reset-list-style;
+  display: flex;
+  align-items: center;
+
+  @include mobileAndBelow {
+    margin-bottom: $spacing-unit / 2;
+  }
+}
+.siteFooter2-links-navItem {
+  & + & {
+    margin-left: $spacing-unit / 4;
+
+    @include mobileAndAbove {
+      margin-left: $spacing-unit / 2;
+    }
+  }
+}
+.siteFooter2-links-navLink {
+  font-family: $tilt2-font-sans;
+  font-size: $tilt2-button-font-size--small;
+  padding-left: $spacing-unit / 4;
+  padding-right: $spacing-unit / 4;
+  height: $tilt2-button-height--small;
+  line-height: $tilt2-button-height--small;
+  border-radius: $tilt2-border-radius;
+  color: $tilt2-color-gray;
+  display: flex;
+  justify-content: center;
+  border: 1px solid $tilt2-color-gray;
+  
+  &:hover {
+    color: $color-white;
+    background-color: $tilt2-color-gray;
+  }
+
+  &--primary {
```

Comment:
I'm surprised this works! tho not sure how i feel about the double-nesting from a clarity/readability perspective

Reply:
Yeah, I think I agree with you. The nesting does get a little involved. I'll un-nest them!